indico/modules/events/templates/display/cern_committees.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'events/display/administrative/event.html' %}
+
+{% block header %}
+    <table class="eventHeader">
+        <tr>
+            <td>
+                <div style="float:left;">
+                    <img src="{{ indico_config.config.ImagesBaseURL }}/images/smallcern.png">
```

Comment:
are you sure the `.config` is needed? if `indico_config` i the thing I added long time ago it should have a `__getattr__` that proxies e.g. `imagesBaseURL` to `getImagesBaseURL()`